apps/website-example/pages/mitmachen.tsx

```diff
@@ -0,0 +1,45 @@
+import {ApiV1, SubscribeContainer} from '@wepublish/website'
+import {NextPageContext} from 'next'
+import getConfig from 'next/config'
+import {ssrAuthLink} from '../src/auth-link'
+import {getSessionTokenProps} from '../src/get-session-token-props'
+
+export default function Mitmachen() {
+  return <SubscribeContainer failureURL="/" successURL="/" />
+}
+
+Mitmachen.getInitialProps = async (ctx: NextPageContext) => {
```

Comment:
Using getInitialProps to preload user data when not already on the site, but not SSRing when the user is already on the site.